### ISSUE TITLE
注入Cycript都成功了，但是打开应用闪退。

### ISSUE THREAD

**Issue by allenbytebuddy** [Urinx/iOSAppHook#36]
请问大家，我经过测试不加CYListenServer(8989);这句打开应用没问题，加了之后打开应用就闪退了，是因为我的IOS版本是10.3.2吗？用了idevicesyslog看到了这种错误 iPhone-5s SpringBoard(KeyboardArbiter)[51] <Error>: HW kbd: Failed to set (null) as keyboard focus，怎么破？或者谁给我发个编译好的cycript.dylib给我试试？不要微信的，就普通的AppDelegate生成的，谢谢了！
